types/color/index.d.ts

```diff
@@ -10,7 +10,9 @@ import convert = require('color-convert');
 
 type ColorParam = Color | string | ArrayLike<number> | number | { [key: string]: any };
 
-interface Color<T extends ColorParam = ColorParam> {
+export interface ColorExtendMethods { }
```

Comment:
This isn't needed - other packages can augment `Color` without this interface

Reply:
How to augment 'Color'  without this interface ? I got the error: error TS2671: Cannot augment module 'Color'

Currently, I used [patch-package](https://github.com/ds300/patch-package) to fix this issue for my project.